Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -282,12 +355,29 @@ def main():
             search_string = demisto.args().get('search_string')
             return_outputs(*Druva_FindDevice_Command(clientObj, search_string))
 
+        if command == 'druva-find-user':
+            user_string=demisto.args().get('user_string')
+            return_outputs(*Druva_FindUser_Command(clientObj,user_string))
```

Comment:
After changing to ComandResults,  lets also change the return outputs to return_results

Reply:
return_outputs is replaced with return_results